CRM/Civicase/Test/Fabricator/CaseCategoryCustomFieldExtend.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * CaseCategoryCustomFieldExtend Fabricator class.
+ */
+class CRM_Civicase_Test_Fabricator_CaseCategoryCustomFieldExtend {
```

Comment:
I don't think we need this fabricator to be honest.
- The Custom Group Extends is not an actual entity and is basically created via the option value API. 

I would suggest we move the function for creating the Custom Group extend to the Test class. If you think we would be needing this functionality in another test, then we should rather create a generic Option Value fabricator similar to https://github.com/compucorp/civihr/blob/master/uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Test/Fabricator/OptionValue.php.
The advantage of this is that the fabricator can be used whenever we need to create an option value by passing some defined parameters and will not be limited to just creating option value for the `cg_extend_objects` option group.

Reply:
Good point. I moved to the test class, as suggested. I did not go for the option of creating the OptionValue fabricator, because I am not aware if we can have more usages later. I will consider the idea for other tickets.